cpu/cc2538/lpm.c

```diff
@@ -92,6 +92,16 @@ rtimer_clock_t lpm_stats[3];
  */
 static rtimer_clock_t sleep_enter_time;
 
+/* CC2538 silicon errata:
+   Possible Incorrect Value of Clock Dividers after PM2 and PM3 */
+#if SYS_CTRL_SYS_DIV == SYS_CTRL_CLOCK_CTRL_SYS_DIV_32MHZ || \
+  SYS_CTRL_IO_DIV == SYS_CTRL_CLOCK_CTRL_IO_DIV_32MHZ
+/* We use this variable in order to flag if we have adjusted either
+   one or both clock divisors upon entering PM2 or PM3. As everything
+   is done at compile time we can use defines to do the remaining stuff */
+static uint8_t reset_div;
+#endif
```

Comment:
This is useless: you only need the values of `SYS_CTRL_SYS_DIV` and `SYS_CTRL_IO_DIV` for the restore stage, and restoring the same values would be harmless anyway. Moreover, the system and I/O clocks must be handled separately.